challenges/lib/src/main/java/challenges/stacksandqueues/PseudoQueue.java

```diff
@@ -0,0 +1,44 @@
+package challenges.stacksandqueues;
+
+import java.util.NoSuchElementException;
+
+public class PseudoQueue<T> {
```

Comment:
You'll need to go back and look at the challenge. You implemented a normal Queue. Instead of using Node<t>, use Stacks.